web-admin/src/routes/[organization]/[project]/+page.svelte

```diff
@@ -1,48 +1,23 @@
 <script lang="ts">
-  import { goto } from "$app/navigation";
   import { page } from "$app/stores";
-  import Button from "@rilldata/web-common/components/button/Button.svelte";
-  import { useDashboardNames } from "@rilldata/web-common/features/dashboards/selectors";
-  import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
-  import { createAdminServiceGetProject } from "../../../client";
+  import ProjectDeploymentLogs from "../../../components/projects/ProjectDeploymentLogs.svelte";
+  import ProjectDeploymentStatus from "../../../components/projects/ProjectDeploymentStatus.svelte";
+  import ProjectGithubConnection from "../../../components/projects/ProjectGithubConnection.svelte";
+  import ShareProjectCta from "../../../components/projects/ShareProjectCTA.svelte";
```

Comment:
Consider using absolute paths

Reply:
Our VSCode settings are: `"typescript.preferences.importModuleSpecifier": "project-relative"`

Think we should change to `shortest` or `non-relative`? Alternatively, Hamilton has proposed SvelteKit aliases.